string              filename
string              file_path
---
bool                              success